api/client/src/main/java/org/projectnessie/client/auth/oauth2/OAuth2Client.java

```diff
@@ -239,7 +232,10 @@ Tokens fetchNewTokens() {
           ClientCredentialsTokensRequest body =
               ImmutableClientCredentialsTokensRequest.builder().scope(scope).build();
           HttpResponse httpResponse =
-              httpClient.newRequest(config.getResolvedTokenEndpoint()).postForm(body);
+              httpClient
```

Comment:
Nit: add a helper method for the `httpClient.newRequest ... postForm()` (it's duplicated often)